tests/contrib/dbapi/test_dbapi.py

```diff
@@ -39,9 +51,10 @@ def test_execute_wrapped_is_called_and_returned(self):
             DD_VERSION="v7343437-d7ac743",
         )
     )
-    def test_curser_execute_with_dbm_injection(self):
+    def test_cursor_execute_with_dbm_injection(self):
```

Comment:
don't this test and test at 531 have the same name? https://github.com/DataDog/dd-trace-py/blob/58d1b172ddf6e51bafe9951cf2d23f287fd97114/tests/contrib/dbapi/test_dbapi.py#L531

Reply:
Yes but they're in different classes. One test is defined in TestTracedCursor and the other is in TestFetchedTracedCursor